@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/XEH_preInit.sqf

```diff
@@ -117,7 +117,7 @@ fza_ah64_salvofired = 0;
 fza_ah64_mynum = 0;
 fza_ah64_lastdir = 0;
 fza_ah64_dps = 0;
-fza_ah64_slip = 0;
+fza_ah64_sideslip = 0;
```

Comment:
Could the renaming of `slip` to `sideslip` go separately please

Reply:
It ensures its the same name across the entire use case

Reply:
I'll leave this one as it is somewhat related